stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/mentions/MentionsFragment.kt

```diff
@@ -25,4 +32,22 @@ class MentionsFragment : Fragment() {
         super.onDestroyView()
         _binding = null
     }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        setupOnClickListeners()
+
+        viewModel.bindView(binding.mentionsListView, viewLifecycleOwner)
+        binding.mentionsListView.setMentionSelectedListener {
+            // TODO add navigation
+            showToast("Selected mention ${it.id}")
+        }
+    }
+
+    private fun setupOnClickListeners() {
+        activity?.apply {
+            onBackPressedDispatcher.addCallback(viewLifecycleOwner) {
+                finish()
```

Comment:
Why do you finish an activity? Why not just popping fragment back stack?

Reply:
For now, I just copied this from `ChannelListFragment`, which has the same behaviour. Could be worth extracting to a common place later on.